src/components/common/Balance.js

```diff
@@ -0,0 +1,46 @@
+import React from 'react'
+
+// denomination of one near in minimal non divisible units (attoNears)
+// NEAR_NOMINATION is 10 ** 18 one unit
+const NOMINATION = 18
+const REG = /(?=(\B)(\d{3})+$)/g;
+
+const Balance = ({amount, milli}) => {
+    if (!amount) {
```

Comment:
shouldn't also check for `milli`? also I'd be explicit and just spell in error that given param is null. So something like "<Balance> `amount` property shouldn't be null`

Reply:
changed